interpreter/package_test.go

```diff
@@ -301,9 +299,6 @@ func TestInterpreter_EvalPackage(t *testing.T) {
 				map[string]values.Value{
 					"x": values.NewInt(10),
 				}),
-			sideEffects: []values.Value{
-				values.NewInt(10),
-			},
```

Comment:
Why does `x = 10` not create a side effect anymore?

Reply:
I thought it was confusing to have all top level statements be considered side effects when really the only statement that should be considered side effect producing is an expression statement.